core/mm/core_mmu.c

```diff
@@ -66,7 +66,7 @@ static struct tee_mmap_region static_mmap_regions[CFG_MMAP_REGIONS
 #if defined(CFG_CORE_ASLR) || defined(CFG_CORE_PHYS_RELOCATABLE)
 						+ 1
 #endif
-						+ 1] __nex_bss;
+						+ 2] __nex_bss;
```

Comment:
This change adds 1, 3 or 4 cells in `static_mmap_regions` (depending on `CFG_NS_VIRTUALIZATION` and `CFG_CORE_RWDATA_NOEXEC`).
Should this be reflected here? Or maybe add 4 cells, unconditionally to cover all cases? Or use explicit config cases (below, not that very helpful I guess)?
```patch
 static struct tee_mmap_region static_mmap_regions[CFG_MMAP_REGIONS
 #if defined(CFG_CORE_ASLR) || defined(CFG_CORE_PHYS_RELOCATABLE)
 						+ 1
 #endif
+#ifdef CFG_CORE_RWDATA_NOEXEC
+#ifdef CFG_NS_VIRTUALIZATION
+ 						+ 4
+#else
+						+ 3
+#endif
+#else
+						+ 1
+#endif /*CFG_CORE_RWDATA_NOEXEC*/
 						+ 1] __nex_bss;
```

Reply:
Yeah, let's do that to keep things simple.